src/components/Form/Fieldset.js

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import styled from "styled-components"
+
+const FieldSet = ({ children, title }) => {
+ return (
+     <Container>
+     <Title>{title}</Title>
+        {children}
+     </Container>
+ )
+}
+
+const Container = styled.form`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    width: 50%;
+    background-color: orange;
+    padding: 1.5em 2em;
```

Comment:
I would actually prefer you to use REM above EM, unless you have a specific reason for it?